### ISSUE TITLE
[Accessibility] ExpenseReport.JSON: Focusable sibling elements must not have same and localized control type.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6267]
### Target Platforms

UWP

### SDK Version

Unknown

### Application Name

Visualizer

### Problem Description

[23703041](https://microsoft.visualstudio.com/OS/_workitems/edit/23703041)

Actual Result:
There is no unique name defined for amount section like(Air Travel Expense $300, Auto Mobile Expense $100) etc. at right pane of Left Pane JSON Files and due to this, screen reader only announce Expand/collapse button which creates confusion for the screen reader user. 

Expected Result:
There should be unique names for sibling controls(Air Travel Expense $300, Auto Mobile Expense $100) etc. that have the same Control type property so that screen reader user could easily identify unique control.

### Card JSON

[ExpenseReport.json](https://github.com/microsoft/AdaptiveCards/blob/main/samples/v1.2/Scenarios/ExpenseReport.json)

**Comment by jwoo-msft** [comment]
closing the bug as it's marked closed at ado
